src/Input/Input.module.scss

```diff
@@ -15,6 +15,13 @@
   height: 32px;
   color: var(--mirage);
 
+  @include responsive('phone-xs', 'phone') {
+    @include text('body-2');
+
+    height: 48px;
+    padding: 12px 16px;
+  }
+
```

Comment:
Maybe we can make it a conditional class? Otherwise these styles will be applied to every input, but we have some them are still `32px` on mobile. My suggestion is to do something like this: `size='mobile-large'` 
![image](https://user-images.githubusercontent.com/25135650/66489039-ee6e5b00-eae9-11e9-90da-e588cf4bff73.png)

Reply:
As I understand, idea was make it comfortable for finger and better to have big variant by default. Nastya didn't rewrite all screens yet